config/initializers/honeycomb.rb

```diff
@@ -0,0 +1,16 @@
+if ENV['APPS_DHH_HONEYCOMB_KEY'].present?
```

Comment:
I think you want this as `APPS_DMR_HONEYCOMB_KEY`

Reply:
@mcritchlow - It has been updated.  Could you review it again?  Thanks